pkg/collector/corechecks/cluster/orchestrator/transformers/k8s/job_test.go

```diff
@@ -270,6 +270,19 @@ func TestExtractJob(t *testing.T) {
 				},
 			},
 		},
+		"job with resources": {
+			input: batchv1.Job{
+				Spec: batchv1.JobSpec{
+					Template: getTemplateWithResourceRequirements(),
+				},
+			},
+			expected: model.Job{
+				Metadata:             &model.Metadata{},
+				Spec:                 &model.JobSpec{},
+				Status:               &model.JobStatus{},
+				ResourceRequirements: getExpectedModelResourceRequirements(),
```

Comment:
Does it make sense to move this function `getExpectedModelResourceRequirements ` from pod_test.go to common_test.go ?

Reply:
hmm I see that this might make sense. It is indeed shared between those resources. But the resource requirements are actually all mapped to the pod template, which maps to pods. So there could be this argument as well. I prefer to keep this here as of now and refactor later. But let me know if you have a strong opinion here